openfecwebapp/api_caller.py

```diff
@@ -86,14 +86,26 @@ def load_legal_advisory_opinion(ao_no):
     if not (documents and len(documents)):
         return None
 
-    canonical_document = documents[0]
+    for document in documents:
+        canonical_document = document
+        if document['category'] == 'Final Opinion':
+            break        
+    
+    if not canonical_document:
```

Comment:
Yep, should never. Just put it there in the event a change breaks:

`if not (documents and len(documents)):`